### PR TITLE
[1.3] Fixed save_statevector() error representing-qubit-states.ipynb

### PR DESCRIPTION
**Pull request by RahulGopan0**
# Changes made
Added code to import Aer explicitly from qiskit.providers.aer

# Justification
Was unable to apply the save_statevector instruction to a newly created QuantumCircuit. Doing so throws an error: AttributeError: 'QuantumCircuit' object has no attribute 'save_statevector'.

### CODE REVIEW

**Comment by frankharkins** [comment]
@RahulGopan0 which version of qiskit are you using? The existing code seems to run fine on the website (qiskit 0.39.0)

**Comment by RahulGopan0** [comment]
@frankharkins The code runs fine on the website. However, if we run the exact same code in a different environment, we receive an error. Although this isn't a big deal, it poses an issue for someone who tries the code in the website voluntarily, so I suggest adding this. 

By the way, I ran the code using qiskit==0.39.2

File: content/ch-states/representing-qubit-states.ipynb [hunk 1]
```diff
@@ -152,7 +152,8 @@
    "source": [
     "from qiskit import QuantumCircuit, assemble, Aer\n",
     "from qiskit.visualization import plot_histogram, plot_bloch_vector\n",
-    "from math import sqrt, pi"
+    "from math import sqrt, pi\n",
+    "from qiskit.providers.aer import Aer"
```
Comment (frankharkins) [inline content/ch-states/representing-qubit-states.ipynb:156]:
```suggestion
    "from qiskit_aer import Aer"
```
I think this is best practice now, see [this comment](https://github.com/qiskit-community/qiskit-textbook/pull/1487#issue-1375669013).

**Review by frankharkins — APPROVED** [review]
Thank you!

**Pull request merged**